profiling/perfetto-connector/Makefile

```
@@ -0,0 +1,18 @@
+
+CXX := g++
+CXXFLAGS := -O3 -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -W -Wall
+
+default: all
+
+all: library strip
+
+library: libperfetto-connector.cpp
```

It is rebuilding each time
```suggestion
all: libperfetto-connector.so strip

libperfetto-connector.so: libperfetto-connector.cpp
```